oauthenticator/oauth2.py

```diff
@@ -1086,3 +1088,20 @@ def __init__(self, **kwargs):
                 self._deprecated_oauth_trait, names=list(self._deprecated_oauth_aliases)
             )
         super().__init__(**kwargs)
+
+
+# patch allowed_users help string to match our definition
+# base Authenticator class help string gives the wrong impression
+# when combined with other allow options
+OAuthenticator.class_traits()[
+    "allowed_users"
+].help = """
```

Comment:
The base Authenticator.allowed_users docstring gives the wrong impression of how it relates to other `allow` config, since the base Authenticator doesn't have other `allow` config.